libcloudforensics/gcp.py

```diff
@@ -865,9 +894,10 @@ def AttachDisk(self, disk, read_write=False):
 
     Args:
       disk (GoogleComputeDisk): Disk to attach.
-      read_write (bool): Boolean saying if the disk should be attached in RW
-      mode.
+      read_write (bool): Optional. Boolean saying if the disk should be
+          attached in RW mode.
```

Comment:
"Optional. Boolean indicating whether the disk should be attached in RW mode. Default is False (read-only)."